### ISSUE TITLE
Intermittent error with Validation in IE 11

### ISSUE THREAD

**Issue by gilles-leblanc** [wet-boew/wet-boew#8148]
Hello, 

From time to time my WET application seems to have some problems with the localization of the validation messages. This is happening intermittently and has so far only been observed with IE11 and not with Chrome or Firefox.

This is the normal output of the validation controls:

> Erreur 1 : Titre de la proposition - Ce champ est obligatoire.

and

> Erreur 1 : Ce champ est obligatoire.

But when the error is happening here is the output:

> undefined 1undefined Titre de la propositionundefinedCe champ est obligatoire.

and

> undefined 1undefined Ce champ est obligatoire.

The last time this error happened I had the Dev Tools in IE 11 opened and saw there was an error in the Javascript:

> Unable to get property 'replace' of undefined or null reference

on this line:

> spanTextFtr = spanTextFtr.replace( "'", "&#39;" );

Of the web-beow.js file. I have traced this to be in wet-beow/overlay.js in the repository more specifically here : https://github.com/wet-boew/wet-boew/blob/d5f465f63f5fc4391038b78fe509268b89bab763/src/plugins/overlay/overlay.js#L75

I hadn't noticed any JS errors in the Dev Tools in the prior occurrences of this error.
I am using this version:  * v4.0.26-development - 2017-06-22

From my quick analysis I can see that the field names and copy material is ok but the punctation and special characters are replaced with *undefined*. spanTextFtr is probably *undefined* when we call the replace method. It is supposed to be assigned here:

```
if ( hasFooter ) {
	spanTextFtr = footer.innerHTML + i18nText.space + i18nText.esc;
} else {
	footer = document.createElement( "div" );
	footer.setAttribute( "class", "modal-footer" );
	spanTextFtr = i18nText.esc;
}
```

The error is happening as I move from field to field. I haven't found any specific repro steps.

**Comment by duboisp** [comment]
What version of WET are you using?  I think that has been fixed in the latest version.

**Comment by duboisp** [comment]
That has been adressed and fixed since ~mid-July. You should not having that issue with the latest release of wet-boew.

**Comment by gilles-leblanc** [comment]
I am using ASP.NET Web Forms so I decided to update my solution to use the latest NuGet package I could use for the GoC.WebTemplate. 

The new NuGet package version is: `4.0.25.5` and it's published date is: August 23, 2017. In the WebConfig the version is `version="v4_0_25"`.

I have these two lines of comment in my page:
`
        `

And finally if I open the CSS file `<link rel="stylesheet" href="https://www.canada.ca/etc/designs/canada/cdts/gcweb/v4_0_25/css/theme.min.css">
` I get the following version number: `v4.0.26-development - 2017-06-22
`.

I am still getting the error:

![image](https://user-images.githubusercontent.com/712010/30611337-0ff91606-9d4f-11e7-8081-9564303ae6c4.png)

And here is another time when I don't have the error:

![image](https://user-images.githubusercontent.com/712010/30611944-08eb9756-9d51-11e7-9e8c-5e1c2bf09c7c.png)

Is there anyway to know if this is because of my code or it's something related with the Wet ? Or maybe something else entirely ?

**Comment by rlambert27** [comment]
@gilles-leblanc For .NET package related to CDTS you should open your ticket here first (http://gitlab.ssc.etg.gc.ca/GOCWebTemplates/DotNetTemplates/issues) so we can eliminate CDTS from the equation. Even though the CDTS CSS file says v4.0.26-development that is because we added a fix shortly after v4.0.25 and so it says we have the development version, but it's still v4.0.25.

Also we released a new version of the CDTS for WET V4.0.26 and the .NET NuGet package will be ready soon, @CalvinRodo can let you know the ETA for that package.

**Comment by gilles-leblanc** [comment]
@rlambert27 Thanks I'll bring it up with them.
